pkgs/os-specific/linux/pam/default.nix

```diff
@@ -26,6 +26,12 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  # Case-insensitivity workaround for https://github.com/linux-pam/linux-pam/issues/569
+  prePatch = lib.optionalString stdenv.buildPlatform.isDarwin ''
+    rm CHANGELOG
+    touch ChangeLog
```

Comment:
I don't understand why `rm CHANGELOG` doesn't fix this on its own. I assumed [autogen.sh](https://github.com/linux-pam/linux-pam/blob/6619819487f9da20a0e6f70aaa92c422d1e6f124/autogen.sh) would have re-created it with the correct case. Oh well.